contrib/ovirt/lib/ovirtlago/repoverify.py

```diff
@@ -107,7 +107,10 @@ def fetch_xml(url):
     Returns:
         lxml.etree._Element: Root of the xml tree for the retrieved resource
     """
-    content = urllib2.urlopen(url).read()
+    try:
+        content = urllib2.urlopen(url).read()
+    except:
+        raise RuntimeError('Failed to open URL %s' % url)
```

Comment:
I don't thin we should swallow the exception, probably you can add the original exception message in the RuntimeError you are throwing so you have both, the url that failed, and the reason why